WARNING:lb-run:Decided best platform to use is x86_64-slc6-gcc49-opt
WARNING:lb-run:Decided best container to use is None
# setting LC_ALL to "C"
# Restarting with LD_PRELOAD='libtcmalloc.so'
# --> Including file '/export/data1/condor/execute/dir_431212/DIRAC_KPfOF9pilot/400864514/AnalysisProductions/dynamic/Lb2Lll/MC_2012_MagDown_Lb2JpsiL_ee_strip_autoconf.py'
# <-- End of file '/export/data1/condor/execute/dir_431212/DIRAC_KPfOF9pilot/400864514/AnalysisProductions/dynamic/Lb2Lll/MC_2012_MagDown_Lb2JpsiL_ee_strip_autoconf.py'
# --> Including file '/export/data1/condor/execute/dir_431212/DIRAC_KPfOF9pilot/400864514/AnalysisProductions/Lb2Lll/stripping_seq.py'
Traceback (most recent call last):
  File "/cvmfs/lhcb.cern.ch/lib/lhcb/GAUDI/GAUDI_v27r0/InstallArea/x86_64-slc6-gcc49-opt/scripts/gaudirun.py", line 425, in <module>
    exec o in g, l
  File "<string>", line 1, in <module>
  File "/nightlies/jenkins-build-release/workspace/nightly-slot-build-platform/build/GAUDI/GAUDI_v27r0/InstallArea/x86_64-slc6-gcc49-opt/python/GaudiKernel/ProcessJobOptions.py", line 415, in importOptions
  File "/nightlies/jenkins-build-release/workspace/nightly-slot-build-platform/build/GAUDI/GAUDI_v27r0/InstallArea/x86_64-slc6-gcc49-opt/python/GaudiKernel/ProcessJobOptions.py", line 387, in _import_python
  File "/export/data1/condor/execute/dir_431212/DIRAC_KPfOF9pilot/400864514/AnalysisProductions/Lb2Lll/stripping_seq.py", line 18, in <module>
    seq = stripping.stripping(data_type, stripping_lines)
  File "/condor/execute/dir_431212/tmpFY6Wza/DBASE/AnalysisProductions/v999999999999/Lb2Lll/helpers/stripping.py", line 41, in stripping
    streams = buildStreams(stripping=strippingConfiguration(stripping_version),
  File "/workspace/build/STRIPPING/STRIPPING_v14r2p1/InstallArea/x86_64-slc6-gcc49-opt/python/StrippingSettings/Utils.py", line 35, in strippingConfiguration
  File "/cvmfs/lhcb.cern.ch/lib/lcg/releases/LCG_83/Python/2.7.9.p1/x86_64-slc6-gcc49-opt/lib/python2.7/shelve.py", line 239, in open
    return DbfilenameShelf(filename, flag, protocol, writeback)
  File "/cvmfs/lhcb.cern.ch/lib/lcg/releases/LCG_83/Python/2.7.9.p1/x86_64-slc6-gcc49-opt/lib/python2.7/shelve.py", line 223, in __init__
    Shelf.__init__(self, anydbm.open(filename, flag), protocol, writeback)
  File "/cvmfs/lhcb.cern.ch/lib/lcg/releases/LCG_83/Python/2.7.9.p1/x86_64-slc6-gcc49-opt/lib/python2.7/anydbm.py", line 84, in open
    mod = __import__(result)
  File "/cvmfs/lhcb.cern.ch/lib/lcg/releases/LCG_83/ROOT/6.06.00/x86_64-slc6-gcc49-opt/lib/ROOT.py", line 301, in _importhook
    return _orig_ihook( name, *args, **kwds )
  File "/cvmfs/lhcb.cern.ch/lib/lcg/releases/LCG_83/Python/2.7.9.p1/x86_64-slc6-gcc49-opt/lib/python2.7/dbhash.py", line 7, in <module>
    import bsddb
  File "/cvmfs/lhcb.cern.ch/lib/lcg/releases/LCG_83/ROOT/6.06.00/x86_64-slc6-gcc49-opt/lib/ROOT.py", line 301, in _importhook
    return _orig_ihook( name, *args, **kwds )
  File "/cvmfs/lhcb.cern.ch/lib/lcg/releases/LCG_83/Python/2.7.9.p1/x86_64-slc6-gcc49-opt/lib/python2.7/bsddb/__init__.py", line 67, in <module>
    import _bsddb
  File "/cvmfs/lhcb.cern.ch/lib/lcg/releases/LCG_83/ROOT/6.06.00/x86_64-slc6-gcc49-opt/lib/ROOT.py", line 301, in _importhook
    return _orig_ihook( name, *args, **kwds )
ImportError: libdb-4.7.so: cannot open shared object file: No such file or directory
